### PR TITLE
new ena-upload tutorial

### PR DESCRIPTION
**Pull request by roncoronimiguel**
hands-on + slides for ena-upload tool

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,14 +0,0 @@
-source "https://rubygems.org"
```
Comment (bgruening) [inline Gemfile:1]: can you please restort the Gemfile and Gemfile.lock? Thanks!

File: Makefile [hunk 2]
```diff
@@ -45,7 +45,7 @@ ACTIVATE_ENV = source $(shell dirname $(dir $(CONDA)))/bin/activate $(CONDA_ENV)
 install: clean create-env ## install dependencies
 	$(ACTIVATE_ENV) && \
 		gem update --no-document --system && \
-		ICONV_LIBS="-L${CONDA_PREFIX}/lib/ -liconv" gem install --no-document addressable:'2.5.2' jekyll jekyll-feed jekyll-scholar jekyll-redirect-from jekyll-last-modified-at csl-styles awesome_bot html-proofer pkg-config kwalify jekyll-sitemap
+		gem install --no-document addressable:'2.5.2' jekyll jekyll-feed jekyll-scholar jekyll-redirect-from jekyll-last-modified-at csl-styles awesome_bot html-proofer pkg-config kwalify jekyll-sitemap
```
Comment (bgruening) [inline Makefile:48]: please revert that commit

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.bib [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+@article{Maier2021,
+  doi = {10.1101/2021.03.25.437046},
+  url = {https://doi.org/10.1101/2021.03.25.437046},
+  year = {2021},
+  month = mar,
+  publisher = {Cold Spring Harbor Laboratory},
+  author = {Wolfgang Maier and Simon Bray and Marius van den Beek and Dave Bouvier and Nathaniel Coraor and Milad Miladi and Babita Singh and Jordi Rambla De Argila and Dannon Baker and Nathan Roach and Simon Gladman and Frederik Coppens and Darren P Martin and Andrew Lonie and Bj\"{o}rn Gr\"{u}ning and Sergei L. Kosakovsky Pond and Anton Nekrutenko},
+  title = {Freely accessible ready to use global infrastructure for {SARS}-{CoV}-2 monitoring}
+}
+
```
Comment (bgruening) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.bib:22]: where is your paper :)

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/slides.html [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+.left[
+Why is raw SARS-CoV-2 sequence data important?
+- Allows reuse of data and reproducibility of analysis
+- Enables discovery of minor allelic variants and [intrahost variation](https://virological.org/t/global-framework-for-sars-cov-2-data-analysis-application-to-intrahost-variation-part-1/623)
+]
+.image-40[
+![Intrahost variation](../../images/Upload-data-to-ENA/intra-host.png)
+]
+
+.reduce70[Minor allelic-variants can be used to detect intrahost variation. From Maier et al., 2001 [doi.org/10.1101/2021.03.25.437046](https://doi.org/10.1101/2021.03.25.437046)]
```
Comment (beatrizserrano) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/slides.html:53]:
```suggestion
.reduce70[Minor allelic-variants can be used to detect intrahost variation. From Maier et al., 2021 [doi.org/10.1101/2021.03.25.437046](https://doi.org/10.1101/2021.03.25.437046)]
```

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Raw reads contain valuable information, such as coverage depth and quality scores, that is lost in a consensus sequence.
+Submission of raw SARS-CoV-2 reads to public repositories allows reuse of data and reproducibility of analysis and enables discovery of minor allelic variants and [intrahost variation]{% cite Maier2021 %}.
+
+The European Nucleotide Archive is an Open and FAIR repository of nucleotide data. As part of the International Nucleotide Sequence Database Collaboration (INSDC), ENA also indexes data from the NCBI and DDBJ {% cite Arita2020 %}. Data submitted to ENA must be accompanied by sufficient metadata. You can learn more from this introductory deck of [slides](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/Upload-data-to-ENA/slides.html) or directly from [ENA](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/metadata.html).
```
Comment (beatrizserrano) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md:55]:
```suggestion
The European Nucleotide Archive is an Open and FAIR repository of nucleotide data. As part of the International Nucleotide Sequence Database Collaboration (INSDC), ENA also indexes data from the NCBI and DDBJ {% cite Arita2020 %}. Data submitted to ENA must be accompanied by sufficient metadata. You can learn more from this [introductory slide deck](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/Upload-data-to-ENA/slides.html) or directly from [ENA](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/metadata.html).
```

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+
+In order to submit data to ENA, you need to have a valid Webin account. If you don't have one already you can register for one [here](https://www.ebi.ac.uk/ena/submit/sra/#registration). Webin credentials need to be included in your Galaxy user information before you can use the {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %}.
+
+> ### {% icon hands_on %} Hands-on: Add Webin credentials to your Galaxy user information
+>
+> 1. If you have not already done so, log in to usegalaxy.eu
+> 2. Navigate to *"User"* > *"Preferences"* on the top menu
+>   - Click on <i class="fa fa-user" aria-hidden="true"></i> **Manage Information**
+>     - Scroll down to *"Your ENA Webin account details"* and fill in your ***ENA Webin ID*** and ***Password***
+>![A textual description of the image](../../images/ENA-credentials.png "ENA Webin Account details ")
```
Comment (beatrizserrano) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md:87]: the description of the image seems to be missing

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+{: .hands_on}
+
+
+# Option 1: submitting to ENA using interactive metadata generator
+
+In this first example you will submit one ONT sequence file using {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %} interactive metadata forms. This method is only convenient for small submissions. For bulk submissions, we recommend you use the metadata template described below in Option 2.
```
Comment (beatrizserrano) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md:93]:
```suggestion
In this first example, you will submit one ONT sequence file using {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %} interactive metadata forms. This method is only convenient for small submissions. For bulk submissions, we recommend you use the metadata template described below in Option 2.
```

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+>    The URL for our example data is this:
+>
+>    ```
+>    https://zenodo.org/api/files/feff876b-15b5-4664-82e7-8bc454d5f7c3/SRR10902284_ONT.fq.gz
+>    ```
+>
+>
+{: .hands_on}
+
+Once the data is uploaded we fill the metadata using {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %}'. Interactive metadata forms are nested to fit [ENA's metadata model](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/metadata.html). Briefly, you add Samples to a Study, Experiments to Samples and Runs to Experiments.
```
Comment (beatrizserrano) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md:110]:
```suggestion
Once the data is uploaded, we fill the metadata using {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %}'. Interactive metadata forms are nested to fit [ENA's metadata model](https://ena-docs.readthedocs.io/en/latest/submit/general-guide/metadata.html). Briefly, you add Samples to a Study, Experiments to Samples and Runs to Experiments.
```

File: topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
...
+
+
+
+
+
+
+
+# Option 2: submitting to ENA using the metadata template
+
+For larger submissions, interactive metadata input can be tedious and not practical. In the second example you will submit two sets of Illumina PE sequence files and input metadata using a [template spreadsheet](https://drive.google.com/file/d/1Gx78GKh58PmRjdmJ05DBbpObAL-3oUFX/view?usp=sharing). For this exercise we provide you with a pre-filled template and encourage you to explore it.
```
Comment (beatrizserrano) [inline topics/galaxy-interface/tutorials/Upload-data-to-ENA/tutorial.md:152]:
```suggestion
For larger submissions, interactive metadata input can be tedious and not practical. In the second example, you will submit two sets of Illumina PE sequence files and input metadata using a [template spreadsheet](https://drive.google.com/file/d/1Gx78GKh58PmRjdmJ05DBbpObAL-3oUFX/view?usp=sharing). For this exercise, we provide you with a pre-filled template and encourage you to explore it.
```

**Review by beatrizserrano — DISMISSED** [review]

**Review by beatrizserrano — DISMISSED** [review]

**Comment by bgruening** [comment]
Oh no, I fixed that already :)

**Comment by bgruening** [comment]
You reverted all my changes :)

**Review by bgruening — APPROVED** [review]

**Pull request merged**